documentation/release_notes.rst

```diff
@@ -8,6 +8,42 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added one more thread parallel backend. It is based on OpenMP 5.0. To enable the OpenMP backend, please define macro ``ONEDPL_USE_OPENMP_BACKEND`` into 1.
```

Comment:
I believe it should be OpenMP* 4.5

Reply:
"Added a new implementation for ``par`` and ``par_unseq`` execution policies based on OpenMP 4.5 pragmas. It can be enabled with the ``ONEDPL_USE_OPENMP_BACKEND`` macro. For more details, see \<a link to the library guide\>"

Reply:
Added requested changes